app/assets/images/bell.svg

```diff
@@ -1 +1 @@
-<svg width="24" height="24" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M10.54 3.652a1.5 1.5 0 012.92 0 7.028 7.028 0 00-2.92 0zM9.5 3.96a2.5 2.5 0 015 0A7.003 7.003 0 0119 10.5v4.626c1.725.444 3 2.01 3 3.874v.5a.5.5 0 01-.5.5h-7.02c-.048.602-.194 1.16-.51 1.603-.427.597-1.097.897-1.97.897s-1.543-.3-1.97-.897c-.316-.443-.463-1.001-.51-1.603H2.5a.5.5 0 01-.5-.5v-.7a3.801 3.801 0 013-3.716V10.5a7.003 7.003 0 014.5-6.54zM6 15h.5a.5.5 0 010 1h-.7A2.8 2.8 0 003 18.8v.2h18a3 3 0 00-3-3h-8a.5.5 0 010-1h8v-4.5a6 6 0 00-12 0V15zm7.477 5h-2.954c.045.452.155.79.321 1.022.199.278.53.478 1.156.478.627 0 .957-.2 1.156-.478.166-.233.276-.57.32-1.022z" /></svg>
\ No newline at end of file
+<svg width="24" height="24" xmlns="http://www.w3.org/2000/svg"><path d="M20 17h2v2H2v-2h2v-7a8 8 0 1116 0v7zm-2 0v-7a6 6 0 10-12 0v7h12zm-9 4h6v2H9v-2z"/></svg>
```

Comment:
Looks like you ran these through https://jakearchibald.github.io/svgomg ? 😄

Reply:
This is so great

Reply:
Every byte counts. 😄

Reply:
@nickytonline yes, i did :) i run every SVGs through SVGOMG and then also manually adjust code if needed. agree: every byte counts :)))